### PR TITLE
add footer and update favicon + metadata of website

### PR DESCRIPTION
**Pull request by KarlAnna**

### CODE REVIEW

**Review by dev1B — CHANGES_REQUESTED** [review]
Can u add a screenshot of the work 
Also, I would like to recommend you isolate styles based on component

**Comment by dev1B** [comment]
example: https://itnext.io/structuring-your-sass-projects-c8d41fa55ed4

**Comment by KarlAnna** [comment]
pr-footer

File: components/Header/Header.tsx [hunk 1]
```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 import Image from 'next/image';
-import './header.scss';
```
Comment (dev1B) [inline components/Header/Header.tsx:3]: You should include styles in[ _app](https://nextjs.org/docs/pages/building-your-application/styling/css-modules)

Comment (dev1B) [inline components/Header/Header.tsx:3]: Should be `import './header.scss';` 
I think "_" does not work in js.

File: next.config.js [hunk 2]
```diff
@@ -1,8 +1,9 @@
 /** @type {import('next').NextConfig} */
 const path = require('path');
+
 const nextConfig = {
   sassOptions: {
-    includePaths: [path.join(__dirname, 'styles')]
```
[the remark this answers is not in the document]
Comment (dev1B) [reply next.config.js:5]: @KarlAnna SCSS folder name, u dont use sass 


https://sass-lang.com/guide/

image
image

File: sass/abstracts/_variables.scss [hunk 3]
```diff
@@ -1,5 +1,8 @@
-/* Font */
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600&family=Yeseva+One&display=swap');
+
+/* Fonts */
 $main-font-family: 'Montserrat', sans-serif;
+$secondary-font-family: 'Yeseva One';
```
Comment (dev1B) [inline sass/abstracts/_variables.scss:5]: Could you add default font, please

File: sass/base/_typography.scss [hunk 4]
```diff
@@ -0,0 +1,19 @@
+body {
+  color: $primary-text-colour;
+  font-family: $main-font-family;
+  // font-weight: 400;
```
Comment (dev1B) [inline sass/base/_typography.scss:4]: comment

**Review by dev1B — CHANGES_REQUESTED** [review]
Could you fix comments, please

File: components/Header/_header.scss [hunk 5]
```diff
@@ -1,34 +1,30 @@
-@import '/app/variables.module.scss';
+@import '/sass/abstracts/variables';
```
Comment (dev1B) [inline components/Header/_header.scss:1]: @import '/scss/abstracts/variables';

File: components/Footer/Footer.tsx [hunk 6]
```diff
@@ -0,0 +1,61 @@
+import Image from 'next/image';
+import Link from 'next/link';
+import './_footer.scss';
```
Comment (dev1B) [inline components/Footer/Footer.tsx:3]: `import './footer.scss';`

File: sass/abstracts/_variables.scss (hunk 3 above)
Comment (KarlAnna) [reply sass/abstracts/_variables.scss:5]: What do you mean by that?

**Review by dev1B — APPROVED** [review]

**Pull request merged**